.changeset/rude-parrots-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/app-basic': patch
+---
+
+Added example of image field support to basic keystone-next example.
```

Comment:
It doesn't need to be removed but just an fyi, changesets are not necessary for things that aren't published like examples.